preset/preset_builtin.list

```diff
@@ -1,9 +1,10 @@
 <resources>
     <section name="PresetTemplate">
-        <integer name="VersionMajor" value="18" />
+        <integer name="VersionMajor" value="19" />
```

Comment:
You didn't change any keys or value types, so a preset version bump isn't necessary